doc/api/http.md

```diff
@@ -3769,8 +3773,9 @@ changes:
     request.
   * `socketPath` {string} Unix domain socket. Cannot be used if one of `host`
     or `port` is specified, as those specify a TCP Socket.
-  * `timeout` {number}: A number specifying the socket timeout in milliseconds.
-    This will set the timeout before the socket is connected.
+  * `timeout` {number}: Socket timeout in milliseconds.
```

Comment:
Make the `timeout` option description consistent with the same option in `http.Agent()` constructor docs on the same page.